modules/rollup-plugin-workbox-inject/index.js

```diff
@@ -15,15 +15,15 @@
  */
 
 const chalk = require('chalk');
-const defaults = require('workbox-build/src/options/defaults');
-const getFileManifestEntries = require('workbox-build/src/lib/get-file-manifest-entries');
-const getManifestSchema = require('workbox-build/src/options/schema/get-manifest');
+const defaults = require('workbox-build/build/options/defaults');
```

Comment:
I realize that we're likely going to rewrite `workbox-build` in TypeScript at some point, so pointing to the `build/` versions for these sub-modules is more likely to be forwards-compatible in terms of loading a CJS module like you'd expect.